### DISCUSSION TITLE
Self hosted doubts!

### DISCUSSION THREAD

**Discussion by maacumba** [flowershow/flowershow#581] [General and Help] [answered]
Dear Flowershow team. I've spent the day dinving into getting to know you better (site, reddit, creators, github).
Thank you for the amazing work. You all seem to be amazing people.

Some things are still now clear to me. My case scenario.

We are a small team of designers working on a small wiki like (or garden like) site. 
- Meaning, we are designers experimenting with different navigation methods. 
The site is maybe closer to a blog (but not really) with mostly text, but also images.
It is hosted on a mini PC in the studio's basement.
We want to have a static website, yet taking advantage of backlinks and wikilinks.

I'm still quite confused... Will Flowershow generate a static website that I can just FTP to my server?

* I'm currently using Obisidian with mkdocs. But I feel mkdocs is maybe too much for our needs. And I love how you guys have the possibility of styling the website fairly easy.

**Comment by rufuspollock** [comment, marked as answer]
> I'm still quite confused... Will Flowershow generate a static website that I can just FTP to my server?

Yes, you just run `npm run export` locally and you have a directory (`out`) you can just upload 😄

**Reply by maacumba** [reply]
Uau, amazing. Looking forward to start prototyping. Thank you.
